Library/Homebrew/test/dev-cmd/bottle_spec.rb

```diff
@@ -227,15 +219,51 @@ def stub_hash(parameters)
       old_spec.sha256("109c0cb581a7b5d84da36d84b221fb9dd0f8a927b3044d82611791c9907e202e" => :catalina)
       old_spec.sha256("7571772bf7a0c9fe193e70e521318b53993bee6f351976c9b6e01e00d13d6c3f" => :mojave)
       new_hash = { "tags" => { "catalina" => "ec6d7f08412468f28dee2be17ad8cd8b883b16b34329efcecce019b8c9736428" } }
+      expected_checksum_hash = { "7571772bf7a0c9fe193e70e521318b53993bee6f351976c9b6e01e00d13d6c3f" => :mojave }
+      expected_checksum_hash[:cellar] = Homebrew::DEFAULT_CELLAR
       expect(homebrew.merge_bottle_spec([:sha256], old_spec, new_hash)).to eq [
         ["sha256 => catalina"],
-        [{ "7571772bf7a0c9fe193e70e521318b53993bee6f351976c9b6e01e00d13d6c3f" => :mojave }],
+        [expected_checksum_hash],
       ]
     end
   end
+
+  describe "::generate_sha256_line", :needs_linux do
```

Comment:
Why `needs_linux`?

Reply:
To speed up the Mac tests. Maybe we could introduce: "run_on_linux_to_not_clutter_the_mac_tests" but I think this is fine.

Reply:
Perhaps I'm wrong but I feel like `generate_sha256_line` shouldn't be that slow; also, not having `needs_linux` makes it easier to test when developing on Mac

Reply:
It's fast. It just the sum of all tests that make things slow. Last time I added tests @MikeMcQuaid asked me to make them linux-only to speed things up. I thought that was a common practice here, but I might be wrong.

Reply:
Oh, okay. I think he was referring to integration tests being slow, but we can wait for him to clarify

Reply:
All integration tests are slow. All non-integration tests are (generally) fast. I think it's fine to remove `:needs_linux` here. Sorry for the confusion!

Reply:
It's Ok, I'll change it. Makes sense.